app.js

```diff
@@ -0,0 +1,23 @@
+
+function isValidCard(cardNumber) {
+  var string = cardNumber.toString();
+  cardNumber !== undefined;
+  cardNumber === parseInt(prompt('Ingrese el número de su tarjeta de crédito'));
+  var cardlength = string.length;
+  var sum = 0;
+  for (var i = 0; i < cardlength; i = i + 2) {
```

Comment:
los dígitos se deben contar de derecha a izquierda, para corregirlo habría que invertir la cadena antes de empezar los ciclos `for`.